src/commands.ts

```diff
@@ -27,12 +27,12 @@ export const commands = async (message: Message) => {
     triggers.find((trigger) => trigger === command)
   ) || { command: fallback };
 
-  if (matching.command === fallback) {
-    message.channel.send(await matching.command(command, embed, message));
-  } else {
-    message.channel.send(
-      await matching.command(args.slice(command.length + 1), embed, message)
-    );
-    cooldown.setCool(message.author.id);
-  }
+  message.channel.send(
+    await matching.command(
+      [command, args.slice(command.length + 1)],
+      embed,
+      message
+    )
+  );
+  cooldown.setCool(message.author.id);
```

Comment:
this might be a bit trickier now with the cooldown feature @RaisinTen. I think we don't want to execute it on the fallback command.
I'm just commenting since I can't review with more attention for a couple of days

Reply:
but everything else looks good 👍

Reply:
@BOLT04 should I revert back to the `if...else` logic or add the `cooldown`s to the end of all other `commandHandler`s? It's alright, you can review this later when you're free. :smiley:

Reply:
I'd like to hear the feedback of more people on this. Maybe we set the cooldown on every command, and it's not a big deal.

What do you think @eddiejaoude and @luckstech ?

Reply:
it would be great to have the cool down on every command 👍

Reply:
Sorry, I've been really busy these last few days.
Looks great to me!